### PR TITLE
fix:  Add repo secret key validate

### PR DESCRIPTION
**Pull request by whDongRui**
* fix: Create user remove default role

* fix: The vxnet list is incomplete

### CODE REVIEW

File: src/config/translation.js [hunk 1]
```diff
@@ -50,6 +50,9 @@ const tsObj = {
     'Runtime Selector missing value': '环境选择器缺失值',
     'Invalid s3 url, should be like s3://s3.pek3a.qingstor.com/op-repo':
       '无效的s3网址，应该是这样: s3://s3.pek3a.qingstor.com/op-repo',
+    'Verify that the information requested is incomplete!': '验证要求输入的信息不完整！',
```
Comment (iwisunny) [inline src/config/translation.js]: `Information to be verified is incomplete`

File: src/locales/zh/translation.json [hunk 2]
```diff
@@ -211,6 +211,7 @@
   "Search Repo": "搜索仓库",
   "Trigger indexer": "触发索引器",
   "Runtime Selectors": "环境选择器",
+  "URL": "网址",
```
Comment (iwisunny) [inline src/locales/zh/translation.json]: `URL`, i think no need to translate.

File: src/pages/Admin/Users/index.jsx [hunk 3]
```diff
@@ -236,7 +236,7 @@ export default class Users extends Component {
       selectName
     } = userStore;
 
-    const data = userStore.users.toJSON();
+    const data = toJS(userStore.users);
```
Comment (iwisunny) [inline src/pages/Admin/Users/index.jsx]: `toJS` is deprecated in mobx, recommend to use `toJSON`

File: src/stores/app/deploy.js [hunk 4]
```diff
@@ -203,7 +203,10 @@ ${this.yamlStr}`;
 
   @action
   async fetchSubnets(runtimeId) {
-    const result = await this.request.get(`clusters/subnets`, { runtime_id: runtimeId });
+    const result = await this.request.get(`clusters/subnets`, {
+      runtime_id: runtimeId,
```
Comment (iwisunny) [inline src/stores/app/deploy.js]: blank line can be removed

File: src/config/translation.js (hunk 1 above)
Comment (whDongRui) [reply src/config/translation.js]: ok.

File: src/locales/zh/translation.json (hunk 2 above)
Comment (whDongRui) [reply src/locales/zh/translation.json]: OK, revome this.

File: src/pages/Admin/Users/index.jsx (hunk 3 above)
Comment (whDongRui) [reply src/pages/Admin/Users/index.jsx]: If use toJSON,  modify user name will also change table list data. Becase observable lead to data bidirectional binding.

**Pull request merged**